subprojects/smoke-test/src/smokeTest/resources/org/gradle/smoketests/gretty-example/build.gradle

```diff
@@ -16,7 +16,7 @@
 
 plugins {
     id "war"
-    id "org.akhikhl.gretty" version "2.0.0"
+    id "org.gretty" version "2.1.0"
```

Comment:
⭕️ We should add the [jacoco plugin](http://akhikhl.github.io/gretty-doc/Code-coverage-support.html) to the smoke test, too. In this way problem with Gretty and `4.6` would have been discovered faster. This doesn't necessarily need to be a part of this PR.

Reply:
That should likely become a new test case. Happy to add it later in a different PR.